features/admin-file.feature

```diff
@@ -0,0 +1,34 @@
+@admin-file @issue-457
+  Feature: An admin user can edit and delete attribute in admin file page
+    As an admin user
+    I can add/edit/save/delete attribute
+
+  Background:
+    Given Gigadb web site is loaded with production-like data
+    When I go to "/adminFile/update/"
+    Then I should see "Login"
```

Comment:
I don’t think these last two statements are required in the `Background` section. They appear to be testing the security of the website to ensure that a guest user is not able to access the admin page for Files. This does not need to be tested before each scenario?

Reply:
These two statements has been removed. And a new scenario has been created to test the security issue in a new commit.